pkg/tsdb/cloudwatch/query_transformer.go

```diff
@@ -101,30 +101,32 @@ func (e *cloudWatchExecutor) transformQueryResponsesToQueryResult(cloudwatchResp
 
 		eq, err := json.Marshal(executedQueries)
 		if err != nil {
-			plog.Error("Could not marshal executedString struct", err)
+			return nil, fmt.Errorf("Could not marshal executedString struct: %w", err)
```

Comment:
Go errors shouldn't be capitalized, also the message looks misleading:
```suggestion
			return nil, fmt.Errorf("could not marshal executed queries %w", err)
```